docs/modules/commons/nav.adoc

```diff
@@ -1,4 +1,5 @@
 .Commons
 * xref:expressions.adoc[Expressions]
 * xref:vividus-steps.adoc[Steps]
+* xref:story-prioritization.adoc[Story Prioritization]
```

Comment:
I think it doesn't make sense to have a separate page, just put the content to Configuration page